### PR TITLE
feat(account): add logout page

### PR DESCRIPTION
**Pull request by Kaskyi**

### CODE REVIEW

File: src/app/account/logout/logout.component.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    public ngOnInit(): void {
+        this.logout();
+    }
+
+    public logout(): void {
+        this.loginService.logout();
+        this.xmEntitySpecWrapperService.clear();
+        this.router.navigate(['']);
+        /** TODO: transform below as a listener of auth state */
+        const body = document.querySelector('body');
```
Comment (Zabavsky) [inline src/app/account/logout/logout.component.ts]: %)

File: src/app/account/logout/logout.component.html [hunk 2]
```diff
@@ -0,0 +1 @@
+<xm-loader [showLoader]="true"></xm-loader>
```
Comment (Zabavsky) [inline src/app/account/logout/logout.component.html]: Why you use here boolean constant instead of something more dynamic?

Comment (Kaskyi) [reply src/app/account/logout/logout.component.html]: I think, for the logout page we need some friendly design. In the current solution its infinite loader

**Comment by Kaskyi** [comment]
this branch was fixed and merged into developer

**Pull request closed without merging**